### PR TITLE
Voting Event API

### LINKED ISSUE #16: Voting event CRUD API
Endpoints:

- Create
- Get
- Get All
- Update
- Delete

### PR DESCRIPTION
**Pull request by benedict-soh**
Closes #16 

If you want another endpoint for voting event API, or if something needs to be updated, please create a new ticket.

### CODE REVIEW

File: flask-api/flaskr/__init__.py [hunk 1]
```diff
@@ -21,17 +21,18 @@ def create_app(test_config=None):
     app.config["DEBUG"] = True
     app.config['JWT_SECRET_KEY'] =  os.environ['FLASK_SECRET_KEY']
     app.config['JWT_TOKEN_LOCATION'] = ['cookies']
-    app.config['JWT_COOKIE_CSRF_PROTECT'] = True
+    app.config['JWT_COOKIE_CSRF_PROTECT'] = False
```
Comment (benedict-soh) [inline flask-api/flaskr/__init__.py:24]: This will need to be updated later whenever @SamTheBosssss is back to help me debug what is going on with this.

Comment (benedict-soh) [reply flask-api/flaskr/__init__.py:24]: Probably in another PR if it requires major changes.

**Review by SamTheBosssss — APPROVED** [review]
Lit

**Review by LittenJV — APPROVED** [review]

**Review by a1719756 — COMMENTED** [review]
Changes look good

**Pull request merged**